oauthlib/oauth2/rfc6749/grant_types/base.py

```diff
@@ -15,13 +16,22 @@
 class GrantTypeBase(object):
     error_uri = None
     request_validator = None
+    default_response_mode = 'fragment'
 
     def create_authorization_response(self, request, token_handler):
         raise NotImplementedError('Subclasses must implement this method.')
 
     def create_token_response(self, request, token_handler):
         raise NotImplementedError('Subclasses must implement this method.')
 
+    def add_token(self, token, token_handler, request):
+        # Only add a hybrid access token on auth step if asked for
+        if not 'token' in request.response_type.split():
```

Comment:
I agree it's instinctively appealing to accept both `token code` and `code token`, but I wonder if that could lead to issues down the road, or conflicts with future versions of the spec?

How do other people feel? @clintonb @thedrow ?